Gemfile

```diff
@@ -9,12 +9,16 @@ gem 'warbler'
 
 gem 'rack-conneg', '~> 0.1.5'
 gem 'rack-rest_api_versioning', '~> 0.0.2'
-gem 'thin'
+#gem 'thin'
 
 gem 'google-api-client'
 
 gem 'ggb', :git => "https://github.com/dlhaines/google_groups_gem.git", :branch => 'TLCPM-343-ruby'
```

Comment:
A released Gemfile should point to a tag, never to master, but since there are no plans to release this we'll let this slide for the time being.